src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -307,6 +313,22 @@ namespace AppInstaller::Repository
                 }
             }
 
+            // Copy tombstoned default sources info
+            for (const auto& userSource : GetSourcesByOrigin(SourceOrigin::User))

```

Comment:
I think it would be better to fix this by adding a flag to request that tombstones be preserved in the Get operation.  Then modifications can work off of that list, thus preserving the tombstones without a need to copy like this.